xml/api/crm_mon-2.15.rng

```diff
@@ -0,0 +1,183 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Shouldn't these both be -2.20.rng?

Reply:
Oh wait, I see what's going on now.  Weird, but yeah I guess that's fine.

Reply:
Right, it's a correction to a past point, so an exception to the usual